### PR TITLE
Feature/lock mutex - Update backend project from movai_classes

### PR DESCRIPTION
**Pull request by MatheusFernandes-Movai**
[FP-885](https://movai.atlassian.net/browse/FP-885)

[FP-885]: https://movai.atlassian.net/browse/FP-885?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: backend/endpoints/api/v1/restapi.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from mimetypes import guess_type
 from string import Template
 from typing import List, Union
+from deprecated.api.models.lock import Lock
```
Comment (Mograbi) [inline backend/endpoints/api/v1/restapi.py:26]: from dal.models import Lock

**Review by Mograbi — CHANGES_REQUESTED** [review]

File: backend/endpoints/api/v1/restapi.py (hunk 1 above)
Comment (MatheusFernandes-Movai) [reply backend/endpoints/api/v1/restapi.py:26]: thank you!

**Review by engdorm — APPROVED** [review]

**Review by quirinpa — APPROVED** [review]

**Review by quirinpa — CHANGES_REQUESTED** [review]
Sorry its drafted and it has conflicts

**Comment by andreparames** [comment]
That has been opened for over two years, closing until it's needed.

**Pull request closed without merging**